pkgs/by-name/go/gokey/package.nix

```diff
@@ -1,16 +1,18 @@
 { lib, buildGoModule, fetchFromGitHub }:
 buildGoModule {
```

Comment:
```suggestion
buildGoModule rec {
```

Missing a rec here, and please squash the commits.

Reply:
Done. Thanks for the review.